src/js/shield_defs.js

```diff
@@ -3107,6 +3107,13 @@ export function loadShields() {
       ))
   );
 
+  // Multistate auto trails
```

Comment:
Placing the multistate auto trails down here means they’ll sort below county and township routes in the legend. I considered putting this section right below the federal shields, but most maps relegate auto trails to the bottom of the legend as miscellany anyways.